WordPress/WordPressTest/MockWordPressOrgXMLRPCApi.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+@objc class MockWordPressOrgXMLRPCApi: NSObject, WordPressOrgXMLRPC {
+
+    var methodPassedIn: String? = nil
+    var parametersPassedIn: [AnyObject]? = nil
```

Comment:
I totally agree.
I saw that scenario but my reasoning for creating that class was to follow the pattern that was being used in other REST tests https://github.com/wordpress-mobile/WordPress-iOS/blob/50c5fdb08358c8325aaf6d7a09cff6bd5e8393ed/WordPress/WordPressTest/MockWordPressComRestApi.swift

I think i'm fine on using this class for simpler tests and `OHHTTPStubs` for complex one but let me know if you think we should just ditch it and use `OHHTTPStubs`